### PR TITLE
Ibm db sa feedstock

### PR DESCRIPTION
**Pull request by amukherjee28**
Checklist
- [ ] Adding support for ibm_db_sa package into conda-forge channel
- [ ] License file is packaged (see [here](https://github.com/amukherjee28/staged-recipes/blob/master/LICENSE.txt)).
- [ ] Source is from official source. https://pypi.org/project/ibm-db-sa/
- [ ] Build number is 1.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://github.com/ibmdb/python-ibmdb) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://github.com/ibmdb/python-ibmdb/blob/master/README.md) before pinging a team.

### CODE REVIEW

**Comment by amukherjee28** [comment]
@conda-forge-admin Do all the automated task need to pass for the pull request to be accepted and feedstock creation?
I see that the linux_64 is failing for glibc check thats the only issue with the pull request now.

**Comment by amukherjee28** [comment]
@conda-forge-admin the linux_64 build fails because of pre - defined installation of glibc version 2.12 and the expectation is to have 2.17
The same things works fine in linux_64_centos7 where the glibc version is 2.17

**Comment by amukherjee28** [comment]
@conda-forge/core the linux_64 build fails because of pre - defined installation of glibc version 2.12 and the expectation is to have 2.17
The same things works fine in linux_64_centos7 where the glibc version is 2.17

**Comment by amukherjee28** [comment]
@conda-forge-admin @conda-forge/core can this be merged and a feedstock be created.

Thanks.

File: recipes/ibm_db_sa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - python
+    - pip
+    - setuptools
+    - libxml2
+
+  run:
+    - python
+    - setuptools
+    - libxml2
+    - sqlalchemy  1.3.23
```
Comment (xhochy) [inline recipes/ibm_db_sa/meta.yaml:32]: Why this this pinned so hard?

File: recipes/ibm_db_sa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+test:
+  imports:
+    - ibm_db_sa
+
+about:
+  home: https://pypi.org/project/ibm-db-sa/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: Python Alchemy driver for DB2 (LUW, zOS, i5) and IDS
```
Comment (xhochy) [inline recipes/ibm_db_sa/meta.yaml:43]: Should read `SQLAlchemy`

File: recipes/ibm_db_sa/conda_build_config.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+cudnn:                                    # [linux64]
+  - undefined                             # [linux64]
+cuda_compiler_version:                    # [linux64]
+  - None                                  # [linux64]
+docker_image:                             # [linux64]
+  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                 # [linux64]
+  - cos7                                  # [linux64]
```
Comment (xhochy) [inline recipes/ibm_db_sa/conda_build_config.yaml:8]: This is not needed, you can add the following to `conda-forge.yml` on the feedstock:

```
os_version:
  linux_64: cos7
```

File: recipes/ibm_db_sa/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [linux64]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (xhochy) [inline recipes/ibm_db_sa/meta.yaml:20]: The packages doesn't seem to contain any compiled code so you probably don't need a compiler.

File: recipes/ibm_db_sa/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aed72b7275ed5327501f179a0302eaf8934e084aeb3c27a7e0b68f1c83f06173
+
+build:
+  number: 1
```
Comment (xhochy) [inline recipes/ibm_db_sa/meta.yaml:13]: Should be ß

File: LICENSE.txt [hunk 6]
```diff
@@ -1,27 +1,55 @@
...
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Apache License
```
Comment (xhochy) [inline LICENSE.txt:1]: Don't change the licence of the staged-recipes repository.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 6 above)
Comment (amukherjee28) [reply LICENSE.txt:1]: @xhochy I have addressed all your comments in a separate pull request 

https://github.com/conda-forge/staged-recipes/pull/14418

Can you please review the pull request.

Thanks

**Pull request closed without merging**